### PR TITLE
adjusting calendar view based on Arnaud's feedback

### PR DESCRIPTION
**Pull request by Tolsa**
- as request, the 2 files of the calendar view with Arnaud's changes
- added an overflow: scroll on CSS of calendar because the scroll did not work on my chrome browser while testing

### CODE REVIEW

File: smart-views/calendar/component.js [hunk 1]
```diff
@@ -16,13 +17,14 @@ export default Ember.Component.extend(SmartViewMixin.default, {
...
+      that.set('calendarId', `${this.get('element.id')}-calendar`);
       Ember.$.getScript('//cdnjs.cloudflare.com/ajax/libs/fullcalendar/3.1.0/fullcalendar.min.js', function () {
         that.set('loaded', true);
 
-        $('#calendar').fullCalendar({
+        $(`#${that.get('calendarId')}`).fullCalendar({
           allDaySlot: false,
           minTime: '00:00:00',
-          defaultDate: new Date(2018, 2, 1),
+          defaultDate: new Date(2018, 5, 1),
```
Comment (SeyZ) [inline smart-views/calendar/component.js]: By default, the date is `today`. We hardcode this date (March 2018) to have enough data when using our Live Demo. Can you revert this change?

File: smart-views/calendar/component.js [hunk 2]
```diff
@@ -76,7 +78,8 @@ export default Ember.Component.extend(SmartViewMixin.default, {
     if (!this.get('records')) { return; }
 
     var events = [];
-    $('#calendar').fullCalendar('removeEvents');
+    var calendar = $(`#${this.get('calendarId')}`);
+  calendar.fullCalendar('removeEvents');
```
Comment (SeyZ) [inline smart-views/calendar/component.js]: Can you double-check the indentation here?

**Review by SeyZ — CHANGES_REQUESTED** [review]

**Comment by Tolsa** [comment]
@SeyZ done, I reverted the date to what it was before + fixed indentation issue
